rekolekcje-engine/src/test/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/ParticipantsAddTest.java

```diff
@@ -71,6 +71,37 @@ public void shouldAddParticipantToSystemWithExistingParticipants() {
     assertThat(allInSystem).contains(addedParticipant);
   }
 
+  @Test
+  public void shouldAddParticipantWithHealthStatusDataAndDataShouldExistInDB(){
+    //given participant with full data in heath status section
+    ParticipantDTO participantWithHealthStatusData = ParticipantsTestData.getSampleParticipantWithHealthReport1;
+
+    //when I ask to save participant with heath status data in db
+    ParticipantDTO addedParticipant = service.save(participantWithHealthStatusData);
+
+    //then participantWithHealthStatusData and addedParticipant should equals excluding id
```

Comment:
jeśli już decydujemy się na takie komentarze w testach, które mają opisywać sekcje given-when-then, to unikałbym używania nazw zmiennych i dosłownego opisywania słowami tego, co poniżej. Po co pisać "zmienna A równa się B", skoro linijkę niżej dokładnie to samo jest napisane. Jedyna wartość takiego komentarza to opisanie sytuacji od strony "biznesowej", np "zostanie zwrócony taki sam uczestnik (w domyśle taki sam jak dodany)", albo "zwrócony uczestnik ma ustawione id" itp.